diff --git a/apps/dashboard/src/App.tsx b/apps/dashboard/src/App.tsx
index 5e92e3c..151f4e2 100644
--- a/apps/dashboard/src/App.tsx
+++ b/apps/dashboard/src/App.tsx
@@ -1,16 +1,16 @@
-import {copyright} from "@monorepo/utils";
-import { useState } from 'react'
-import logo from './logo.svg'
-import './App.css'
+import { copyright } from '@monorepo/utils';
+import { useState } from 'react';
+import logo from './logo.svg';
+import './App.css';
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [count, setCount] = useState(0);
 
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
-        <p>Hello Vite + React! {copyright}</p>
+        <p>Dashboard Application! {copyright}</p>
         <p>
           <button type="button" onClick={() => setCount((count) => count + 1)}>
             count is: {count}
@@ -40,7 +40,7 @@ function App() {
         </p>
       </header>
     </div>
-  )
+  );
 }
 
-export default App
+export default App;
diff --git a/apps/dashboard/src/main.tsx b/apps/dashboard/src/main.tsx
index 4a1b150..9aa52ff 100644
--- a/apps/dashboard/src/main.tsx
+++ b/apps/dashboard/src/main.tsx
@@ -1,10 +1,10 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App'
-import './index.css'
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App';
+import './index.css';
 
 ReactDOM.createRoot(document.getElementById('root')!).render(
   <React.StrictMode>
     <App />
-  </React.StrictMode>
-)
+  </React.StrictMode>,
+);
diff --git a/libs/utils/src/index.ts b/libs/utils/src/index.ts
index ea601e7..8ffa5e1 100644
--- a/libs/utils/src/index.ts
+++ b/libs/utils/src/index.ts
@@ -1 +1,3 @@
-export const copyright: string = "egghead.io";
+export const copyright: string = 'egghead.io';
+
+const currentYear = new Date().getFullYear();
